server/views/components/sample/sample.njk

```diff
@@ -2,6 +2,6 @@
   <div class="sample-component">
     <h2 class="sample-component__heading">Sample component</h2>
     <p>{{ text }}</p>
-    {% include "views/components/grid-placeholder.njk" %}
+    {% include "views/grid/placeholder/placeholder.njk" %}
```

Comment:
The sub directories allow us to add README.md and config files related to the components, which are utilised in the styleguide. We could do it without sub directories, but I think it helps with finding things.  What complications does it cause?